config/jobs/kubernetes/sig-release/release-branch-jobs/1.27.yaml

```diff
@@ -79,7 +79,7 @@ periodics:
           cpu: "1"
           memory: 3Gi
 - annotations:
-    fork-per-release-periodic-interval: 24h
+    fork-per-release-periodic-interval: ""
```

Comment:
this is getting dropped... intentional? or bug?

Reply:
is it because this is the oldest stable branch?

Reply:
> is it because this is the oldest stable branch?

Yes, exactly! This annotation contains a list of annotations to be used when forking and rotating this job. Because this job is not getting forked/rotated after this release, it doesn't have any interval left.